src/main/kotlin/org/openrewrite/Licenses.kt

```diff
@@ -45,7 +45,12 @@ fun getLicense(recipeOrigin: RecipeOrigin): License {
 
     if (recipeOrigin.groupId == "org.openrewrite") {
         return License.Apache2
+    } else if (recipeOrigin.groupId == "io.moderne.recipe") {
+        return License.Proprietary
+    } else if (recipeOrigin.groupId == "org.openrewrite.recipe" && recipeOrigin.artifactId == "rewrite-java-security") {
+        return License.Proprietary
```

Comment:
Does this make sense? Or are we going to have too many one-of if statements like this if we go with this approach?